src/pages/Login/index.tsx

```diff
@@ -119,7 +120,41 @@ const LoginPage = ({ type }: ILoginPage): JSX.Element => {
       content = (<BusinessForm onSubmit={onSubmit}/>);
       break;
     case 'login':
-      content = null;
+      onSubmit = async (data: LoginFormData) => {
+        const businessId = 1234;
+        signInWithEmailAndPassword(auth, data.Email, data.Password)
+          .then(async (userCredential) => {
+            // Signed in
+            const user = userCredential.user;
+            const reqData = {
+              firebaseId: user.uid,
+              email: data.Email,
+              password: data.Password,
+              businessId,
+              root: true, // True only while creating the business
+            };
+            await fetch(`/business/${businessId}/user`, {
+              method: 'POST',
+              headers: {
+                'Content-Type': 'application/json',
+              },
+              body: JSON.stringify(reqData),
```

Comment:
Why the POST request? You are not creating any user. You are simply allowing an already created user to get his information and give access to the dashboard. A GET request will be enough here. You can use the [API Document](https://documenter.getpostman.com/view/13404327/2s83zcSmZF#3fcdc48f-adf2-4529-b5fe-a4f10b4b339e) to get information for the correct endpoint. Since the endpoint to get a specific employee is not yet created, maybe use the `getAllEmployee` for now then filter based on the firebaseId.

*** Make sure to add a bug issue that make mention of this since this is not good practice and it's actually not safe. But for now it will do since the endpoint doesn't exist ***